### PR TITLE
Update docs around what image formats are supported by GHCR

### PR DESCRIPTION
**Pull request by markphelps**
GHCR supports:

* Docker v2 image spec
* OCI image spec
* Docker manifest lists/OCI image indexes

Screen Shot 2020-10-26 at 10 14 52 AM

### Why:

* The info around what image types supported by GH Container Registry (ghcr.io) is incorrect

### What's being changed:

* The info around what image types supported by GH Container Registry

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/packages/getting-started-with-github-container-registry/about-github-container-registry.md [hunk 1]
```diff
@@ -8,19 +8,24 @@ versions:
 
 {% note %}
 
-**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. Currently, {% data variables.product.prodname_github_container_registry %} only supports Docker image formats. During the beta, storage and bandwidth is free.
+**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. During the beta, storage and bandwidth is free.
```
Comment (markphelps) [inline content/packages/getting-started-with-github-container-registry/about-github-container-registry.md:11]: should this say:

> storage and bandwidth _are_ free.

?

**Comment by janiceilene** [comment]
@markphelps Thanks for opening a PR! I've triaged this for review.

**Review by shayneburgess — APPROVED** [review]

**Comment by markphelps** [comment]
@janiceilene thanks! is this still queued for review?

**Comment by janiceilene** [comment]
It definitely is @markphelps! Thanks for your patience 💛 We're still working our way through all of the wonderful contributions from Hacktoberfest.

File: content/packages/getting-started-with-github-container-registry/about-github-container-registry.md (hunk 1 above)
Comment (runleonarun) [inline content/packages/getting-started-with-github-container-registry/about-github-container-registry.md:11]:
```suggestion
**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. During the beta, storage and bandwidth are free.
```

**Review by runleonarun — COMMENTED** [review]
@markphelps This looks good! I just wonder if we also need to update this note
Screen Shot 2020-11-03 at 5 48 36 PM

https://docs.github.com/en/free-pro-team@latest/packages/publishing-and-managing-packages/about-github-packages#about-billing-for-github-packages

File: data/reusables/package_registry/container-registry-beta.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% note %}
 
-**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. Currently, {% data variables.product.prodname_github_container_registry %} only supports Docker image formats. During the beta, storage and bandwidth is free. For more information, see "[About {% data variables.product.prodname_github_container_registry %}](/packages/getting-started-with-github-container-registry/about-github-container-registry)."
+**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. During the beta, storage and bandwidth is free. For more information, see "[About {% data variables.product.prodname_github_container_registry %}](/packages/getting-started-with-github-container-registry/about-github-container-registry)."
```
Comment (runleonarun) [inline data/reusables/package_registry/container-registry-beta.md:3]:
```suggestion
**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. During the beta, storage and bandwidth are free. For more information, see "[About {% data variables.product.prodname_github_container_registry %}](/packages/getting-started-with-github-container-registry/about-github-container-registry)."
```

**Review by runleonarun — COMMENTED** [review]
Oh here's another "is" that should be "are"

File: data/reusables/package_registry/container-registry-beta.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% note %}
 
-**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. Currently, {% data variables.product.prodname_github_container_registry %} only supports Docker image formats. During the beta, storage and bandwidth is free. For more information, see "[About {% data variables.product.prodname_github_container_registry %}](/packages/getting-started-with-github-container-registry/about-github-container-registry)."
+**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. During the beta, storage and bandwidth are free. For more information, see "[About {% data variables.product.prodname_github_container_registry %}](/packages/getting-started-with-github-container-registry/about-github-container-registry)."
```
Comment (rachmari) [inline data/reusables/package_registry/container-registry-beta.md:3]: Same suggestion as ☝️ 

```suggestion
**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. Currently, {% data variables.product.prodname_github_container_registry %} only supports Docker image and Open Container Initiative (OCI) image formats. During the beta, storage and bandwidth are free. For more information, see "[About {% data variables.product.prodname_github_container_registry %}](/packages/getting-started-with-github-container-registry/about-github-container-registry)."
```

File: content/packages/getting-started-with-github-container-registry/about-github-container-registry.md (hunk 1 above)
Comment (rachmari) [reply content/packages/getting-started-with-github-container-registry/about-github-container-registry.md:11]: Yep! Let's update that. 👍 

```suggestion
**Note:** {% data variables.product.prodname_github_container_registry %} is currently in public beta and subject to change. Currently, {% data variables.product.prodname_github_container_registry %} only supports Docker image and Open Container Initiative (OCI) image formats. During the beta, storage and bandwidth is free.
```

**Review by rachmari — COMMENTED** [review]
@markphelps I chatted with the product folks about the support. What do you think about keeping the note but expanding it to include the image portion of the OCI spec? I think the list you added to the "Supported formats" section clarifies this even further.

**Comment by markphelps** [comment]
> @markphelps I chatted with the product folks about the support. What do you think about keeping the note but expanding it to include the image portion of the OCI spec? I think the list you added to the "Supported formats" section clarifies this even further.

Sounds good to me, I just thought that sense they are really the only image specs possible that it was redundant, but I'm good either way 👍🏻

**Comment by rachmari** [comment]
@markphelps 
> Sounds good to me, I just thought that since they are really the only image specs possible that it was redundant, but I'm good either way 👍🏻

🙈 No you're right. I removed those suggestions and I'll get this shipped as-is!

**Pull request merged**